totalRP3/Modules/NamePlates/NamePlates_Plater.lua

```diff
@@ -0,0 +1,355 @@
+	-- should be sanity checked on OnModuleEnable to make sure someone else
+	-- didn't trample over it.
+
+	TRP3_NAMEPLATES_ADDON = "Plater";
+end
+
+function TRP3_PlaterNamePlates:OnModuleEnable()
+	-- Sanity check TRP3_NAMEPLATES_ADDON to ensure that we remain the chosen
+	-- decorator addon, since it may get trampled by external code.
+
+	if TRP3_NAMEPLATES_ADDON ~= "Plater" then
+		return false, L.NAMEPLATES_MODULE_DISABLED_BY_EXTERNAL;
+	end
+
+	TRP3_NamePlates.RegisterCallback(self, "OnNamePlateDataUpdated");
+
+	TRP3_API.Events.registerCallback("CONFIGURATION_CHANGED", function(...) return self:OnConfigurationChanged(...); end);
+
+	self.unitDisplayInfo = {};
+	self.initialized = {};
+end
+
+function TRP3_PlaterNamePlates:OnConfigurationChanged(_)
+	self:UpdateAllNamePlates()
+end
+
+function TRP3_PlaterNamePlates:OnNamePlateDataUpdated(_, nameplate, unitToken, displayInfo)
+	self:SetUnitDisplayInfo(unitToken, displayInfo);
+	if nameplate.Plater then
+		self:UpdateNamePlate(nameplate);
+	end
+end
+
+function TRP3_PlaterNamePlates:OnNamePlateCreate(nameplate)
+	local nameplateKey = nameplate.unitFramePlater.namePlateUnitToken;
+
+	if self.initialized[nameplateKey] then
+		-- For safety, should be impossible.
+		return;
+	end
+
+	hooksecurefunc(nameplate.unitFramePlater, "UpdateName", function(...) return self:OnNameplateNameTextUpdated(...); end);
+	hooksecurefunc(nameplate.unitFramePlater, "UpdateHealthColor",
+		function(...) return self:UpdateNamePlateHealthBar(...); end);
+
+	hooksecurefunc(Plater, "UpdateUnitName",
+		function(...) return self:UpdateNamePlateNameText(...); end);
+
+	do
+		-- Icon widget.
+		if nameplate.actorType == "friendlyplayer" then
+			local iconWidget = nameplate:CreateTexture("TRP3PlaterIcon", "ARTWORK");
+			iconWidget:ClearAllPoints();
+			iconWidget:SetPoint("RIGHT", nameplate.CurrentUnitNameString, "LEFT", -4, 0);
+			iconWidget:Hide()
+
+			nameplate.unitFramePlater.TRP3PlaterIcon = iconWidget;
+		end
+	end
+	do
+		-- Title text widget.
+		if nameplate.actorType == "friendlyplayer" then
+			local titleWidget = nameplate:CreateFontString("TRP3PlaterTitle", "ARTWORK");
+			titleWidget:ClearAllPoints();
+			titleWidget:SetPoint("TOP", nameplate.CurrentUnitNameString, "BOTTOM", 0, -2);
+			titleWidget:SetTextColor(1, 1, 1, 0.8);
+			titleWidget:SetShadowOffset(1, -1);
+			titleWidget:SetShadowColor(0, 0, 0, 1);
+			titleWidget:Hide()
+
+			nameplate.unitFramePlater.TRP3PlaterTitle = titleWidget;
+		end
+	end
+
+	self.initialized[nameplateKey] = true;
+end
+
+function TRP3_PlaterNamePlates:AppendGuildNameToNameString(name, nameplate)
+	if nameplate.playerGuildName then
+		if nameplate.PlateConfig.show_guild_name then
+			return name .. "\n" .. "<" .. nameplate.playerGuildName .. ">"
+		else
+			return name
+		end
+	else
+		return name
+	end
+end
+
+function TRP3_PlaterNamePlates:AppendFullTitleToNameString(name, fullTitle)
+	return name .. "\n" .. "<" .. fullTitle .. ">"
```

Comment:
From recollection neither of the existing decorators for nameplates surround the full title text in "<>". I think I'd gone with that due to the (very low) default cropping limit for nameplates and to cut down on visual noise. Not necessarily _opposed_ to Plater shoving them in, but might be worth seeing how they look without.

Reply:
I went ahead and removed the "<>". Feels better suited for the guild name and creates unnecessary noise around the title that hurts readability. (not to mention the cropping limit)